### ISSUE TITLE
Fix styling for buttons: background should be white, not transparent

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#9610]
## What happened?

Found during the Design Council review. We are using the secondary button styling for the Yes/No buttons. The default state background should be white, not transparent.

https://design.va.gov/components/buttons

It's hard to see on the above web page since the white buttons are sitting on a white background already. 

## Acceptance Criteria
- [ ]  Validate that the button background is white

**Comment by artsymartha68** [comment]
FYI @Yuffster-usds in case you're digging into CSS near the button styling....

**Comment by mchelen-gov** [comment]
Ah, it looks like the borderline contrast ratio is due to a combination of:
- https://design.va.gov/components/buttons
- https://design.va.gov/components/featured-content

From the perspective of the design guide, are we not supposed to be using buttons within a featured content component, or should their button and feature component styles be adjusted?

Interestingly, the "What's new?" button on https://design.va.gov itself uses the `site-button-inverse` class which may come from https://designsystem.digital.gov/components/button/ but is not explicitly mentioned on https://design.va.gov/components/buttons

**Comment by artsymartha68** [comment]
So per https://design.va.gov/components/buttons
the background is indeed transparent. 

But nothing states it must appear on a white background. So how does one override the default style and make it white?

<img width="355" alt="Screenshot 2020-06-12 12 03 37" src="https://user-images.githubusercontent.com/47792423/84522592-ec762080-aca4-11ea-8561-61861ae22e51.png">

cc @Clark-W-Griswold for our meeting later today.

**Comment by jenstrickland** [comment]
It is my understanding that the design system secondary button background being transparent is a design system bug. 

@emilywaggoner @1Copenut @rianfowler — can you provide accurate guidance on this, please?

**Comment by Yuffster-usds** [comment]
Fixed in https://github.com/department-of-veterans-affairs/vets-website/commit/6509047ef147af4b9023a9d29cdbe58deef90146

**Comment by batemapf** [comment]
https://github.com/department-of-veterans-affairs/vets-website/pull/13186

**Comment by mchelen-gov** [comment]
alternate PR ready for review: https://github.com/department-of-veterans-affairs/vets-website/pull/13212

**Comment by mchelen-gov** [comment]
@artsymartha68 based on https://design.va.gov/documentation/updating-this-site it seems like we are supposed to open a bug report in https://github.com/department-of-veterans-affairs/vets-design-system-documentation to get the design system bug resolved

**Comment by artsymartha68** [comment]
Design system issue created above. I think we have at least 3 issues on the topic, hopefully it'll get fixed!  :)
